packages/components/src/Form/Fields/FieldInline.tsx

```diff
@@ -0,0 +1,151 @@
+  ValidationMessage,
+  ValidationMessageProps,
+} from '../ValidationMessage/ValidationMessage'
+
+export interface CustomizableFieldAttributesInterface
+  extends CustomizableAttributes {
+  labelMargin: SpacingSizes
+}
+
+export interface FieldInlineProps {
+  className?: string
+  disabled?: boolean
+  /**
+   * Defines the label for the field.
+   */
+  label?: string
+  /**
+   * Specifies the fontWeight of the internal Label.
+   * TODO - Deprecate usage in HT, then here.
+   */
+  labelFontSize?: FontSizes
+  /**
+   * Specifies the fontWeight of the internal Label.
+   */
+  labelFontWeight?: FontWeights
+  /**
+   * Whether or not the field should display a `*` denoting it is required.
+   */
+  required?: boolean
+  /**
+   *
+   * Holds the type of validation (error, warning, etc.) and corresponding message.
+   */
+  validationMessage?: ValidationMessageProps
+}
+
+const RequiredStar = styled((props) => (
+  <span {...props} aria-hidden="true">
+    {' '}
+    *
+  </span>
+))`
+  color: ${(props) => props.theme.colors.palette.red500};
+`
+
+/**
+ * `<Field />` allows the rendering of a label (optionally associated with a child input like `<InputText />`),
+ * and can render a validation message. Generally, this component is used with form inputs to give user
+ * feedback about the status of the input values.
+ */
+
+const FieldInlineLayout: FunctionComponent<FieldInlineProps> = ({
+  className,
+  children,
+  label,
+  labelFontSize,
+  labelFontWeight,
+  required,
+  validationMessage,
+}) => {
+  return (
+    <label className={className}>
+      <Label as="span" fontWeight={labelFontWeight} fontSize={labelFontSize}>
+        {label}
+        {required && <RequiredStar />}
+      </Label>
+      <InputArea>{children}</InputArea>
+      <MessageArea>
+        {validationMessage ? (
+          <ValidationMessage {...validationMessage} />
+        ) : null}
+      </MessageArea>
+    </label>
+  )
+}
+
+/**
+ * TODO:
+ *
+ * FieldInline grid layout  - ? why there is extra space on grid around "icon"
+ * Disable validation on FieldRadio (that's silly) - done
+ * Radio, ToggleSwitch & Checkbox disabled style - Label isn't respond to style when disabled
+ * All Input should have red border on error
+ * FieldTextArea label position
+ *
+ */
+
+const InputArea = styled.div``
+const MessageArea = styled.div``
+
+export const FieldInline = styled(FieldInlineLayout)`
+  align-items: center;
+  display: grid;
+  grid-template-areas: 'input label' '. messages';
+  grid-template-columns: repeat(3, max-content);
+
+  ${InputArea} {
+    grid-area: input;
+  }
+
+  ${Label} {
+    grid-area: label;
+    font-size: ${(props) => props.theme.fontSizes.small};
+    font-weight: normal;
+    padding-left: ${(props) => props.theme.space.xsmall};
+
+    color: ${({ theme, disabled }) =>
+      disabled && theme.colors.palette.charcoal500};
+  }
+
+  ${MessageArea} {
+    grid-area: messages;
+    padding-left: ${(props) => props.theme.space.small};
```

Comment:
Destructure theme from props for consistency with other call sites in this file

```suggestion
    padding-left: ${({ theme }) => theme.space.small};
```